File: email-services/Models/EmailMessage.cs
namespace email_services.Models;

public class EmailMessage
{
    public string Uid { get; set; }
    public string? Subject { get; set; }
    public string? From { get; set; }
    public DateTime Date { get; set; }
    public string? BodyPreview { get; set; } 
    public string? BodyHtml { get; set; }    
    public string? BodyText { get; set; }    
    public bool Read { get; set; }
    public bool Starred { get; set; }
}

File: OrgaFlow.UI/orga-flow-ui/src/types/EmailMessage.ts
// Create or update the EmailMessage interface to include all required properties
export interface EmailMessage {
    uid: string; 
    subject?: string;
    from?: string;
    date: string; 
    bodyPreview?: string;
    bodyHtml?: string; 
    bodyText?: string; 
    read: boolean;
    starred?: boolean;
}

